components/schema-migrator/main.go

```diff
@@ -98,9 +143,32 @@ func invokeMigration() error {
 		return fmt.Errorf("# COULD NOT ESTABLISH CONNECTION TO DATABASE WITH CONNECTION STRING: %w", err)
 	}
 
+	log.Println("# STARTING COPY MIGRATION FILES #")
+
+	var fs FileSystem = osFS{}
+	migrationTempPath := fmt.Sprintf("tmp-migrations-%s-*", os.Getenv("MIGRATION_PATH"))
+
+	tmpDir, err := os.MkdirTemp("/migrate", migrationTempPath)
+	if err != nil {
+		return fmt.Errorf("# COULD NOT CREATE TEMPORARY FILE FOR MIGRATION: %w", err)
+	}
+	defer os.RemoveAll(tmpDir)
+
+	migrationNewPath := fmt.Sprintf("new-migrations/%s", os.Getenv("MIGRATION_PATH"))
```

Comment:
More often the terms `destination` and `source` are used.

Reply:
`destination` and `source` are not appropriate for this case because `migrationNew` and `migrationOld` are source directories. I need to figure out something else.

Reply:
:+1: